lib/store.js

```diff
@@ -37,6 +74,18 @@ class CpuPlayer extends Player {
     this.name = "CPU";
     this.isHuman = false;
   }
+  _generateCpuGridChoice() {
+    return Math.floor(Math.random() * 9 + 1);
+  }
+  getGridChoice(store) {
```

Comment:
YES. This is the shape I was after.

Reply:
it doesn't matter at the moment, but it's traditional to make both implementations have the same signature (the other one is `async` so it returns a promise. When you port this to typescript, it will probably want this one to be an async function as well, even though `await 1` behaves the same as `await Promise((resolve) => resolve(1))`)